rotkehlchen/api/v1/resources.py

```diff
@@ -363,7 +364,7 @@ def put(
             self,
             settings: ModifiableDBSettings,
     ) -> Response:
-        return self.rest_api.set_settings(settings)
+        return self.rest_api.set_settings(settings=settings)
```

Comment:
why do all of them need to set the kwargs? Or you just did it? If is only one argument there is no place to confusion and is fine afaik

Reply:
Just to have more consistent style. Async endpoints now have to use kwargs, so I thought that to stay in line it will make sense to have all the endpoints use kwargs.

Reply:
But then is not style, is indeed needed or otherwise the call will fail. Mmmmm this looks like a very easy to write bugs :sweat:

Reply:
This should not cause bugs since if the arguments will be passed as args, the `rest.py`'s method will not even be executed, but rather will explicitly fail.

The `asynchronous` decorator accepts only kwargs, so if the arguments will be passed as args, the `rest.py`'s method will not receive enough arguments for the execution.